contents/handbook/engineering/development-process.md

```diff
@@ -1,54 +1,27 @@
 ---
-title: Shipping things, step by step
+title: Shipping & Releasing
```

Comment:
lowercase "releasing"! :-P

Reply:
Also, should the title more closely match the slug?

Reply:
```suggestion
title: Shipping & releasing
```